code/model/ErrorPage.php

```diff
@@ -274,47 +301,65 @@ public function fieldLabels($includerelations = true) {
 		
 		return $labels;
 	}
-	
+
 	/**
-	 * Returns an absolute filesystem path to a static error file
-	 * which is generated through {@link publish()}.
-	 * 
-	 * @param int $statusCode A HTTP Statuscode, mostly 404 or 500
-	 * @param String $locale A locale, e.g. 'de_DE' (Optional)
+	 * Get error content for the given status code. Only returns pre-cached content
+	 * stored either in the asset store or in the static path (if enabled).
 	 *
+	 * @param int $statusCode A HTTP Statuscode, typically 404 or 500
 	 * @return string
 	 */
-	public static function get_filepath_for_errorcode($statusCode, $locale = null) {
-		if (singleton('ErrorPage')->hasMethod('alternateFilepathForErrorcode')) {
-			return singleton('ErrorPage')-> alternateFilepathForErrorcode($statusCode, $locale);
```

Comment:
Did you remove `$locale` support on purpose? Subsites uses it for (optional) translatable support: https://github.com/silverstripe/silverstripe-subsites/blob/9c9e0bfa94a568e54b37b431529293da09bf8eb9/code/extensions/ErrorPageSubsite.php#L31

I guess it's only one of many variants which could influence the error page content and presentation, so in general I agree we shouldn't pass it through here as a parameter. The extension hooks can get all the info they need from global state anyway (current subsite, current locale). Just making sure it's been actively considered.